### ISSUE TITLE
Passing IP address as argument to 'server:status' returns argument error

### ISSUE THREAD

**Issue by monocrstl** [symfony/symfony#28777]
**Symfony version(s) affected**: 4.1.6

**Description**  
I was reading [How to Use PHP'S built-in Web Server (Symfony Docs)](https://symfony.com/doc/current/setup/built_in_web_server.html) documentation page and in the **Starting the Web Server** section there's a box that shows how the `server:status` command works. It says that we can pass an IP address and a port as a command-line argument, like this: `php bin/console server:status 192.168.0.1:8080`. When I did that (I had the server running on that IP address and port) I got an error message saying that I passed too many arguments. It expects only one argument (`command`).

**How to reproduce**  
```bash
# Install web-server-bundle as dev dependency
$ composer require symfony/web-server-bundle --dev

# Start server with server:start
$ php bin/console server:start 192.168.0.1:8080
# => [OK] Server listening on http://192.168.0.1:8080

# Run server:status for 192.168.0.1:8080
$ php bin/console server:status 192.168.0.1:8080
# => Too many arguments, expected arguments "command".
```

**Comment by xabbuh** [comment]
Thank you for reporting. In fact, the documentation is outdated here. As this is no longer possible since the introduction of the WebServerBundle. See symfony/symfony-docs#10455 and #28778 for PRs updating it.

**Comment by monocrstl** [comment]
@xabbuh Same thing happens for `server:stop <ip>`.

**Comment by xabbuh** [comment]
@monocrstl Thanks for noticing, this will be fixed in symfony/symfony-docs#10455 too.

**Comment by javiereguiluz** [comment]
@monocrstl thanks for reporting this problem and we're sorry for this issue. As @xabbuh said, this was really a documentation problem, so we've updated it in https://github.com/symfony/symfony-docs/pull/10455 and the new doc will be online soon, after it's regenerated. Thanks!
